### PR TITLE
Scrollbar styling changed

### LINKED ISSUE #82: The scrollbar is not very attractive
### Description of the enhancement. ✍🏻

The website is beautiful but the scrollbar is very simple.
I can add styling to the scrollbar so that it looks more attractive and the website also looks better.
I can change the styling of scrollbar .Can you assign this issue to me?

### Describe briefly how you would implement this enhancement.

```bash
Some steps to implement this enhancement-
1. Step 1-I will change the color of scrollbar 
2. Step 2-Adjust its width
3. Step 3-Change its color on hovering
```

### Screenshots Example (if applicable)

_No response_

### Additional information

_No response_

### Contributor terms

- [X] I agree to follow the Contributing Instructions

### PR DESCRIPTION
**Pull request by Shreyans2004**
## Related Issue(s)

Closes  #82
## Proposed Changes
I have added the code for styling of scrollbar. Can it be merge now?

## Type of Change

- [ ] New Feature (Landing Page)
- [ ] New Feature (App)
- [ ] Bug (Landing Page)
- [ ] Bug (App)
- [ ] New Component (App)
- [ ] Enhancement
- [ ] Documentation / ReadMe / Workflows
- Other (If not in above options):  

## Your Idea for Level (GSSoC)

- [ ] Level 1
- [x] Level 2
- [ ] Level 3

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.11.2",
-    "react-scripts": "5.0.1"
+    "react-scripts": "^5.0.1"
```
Comment (praveenscience) [inline package.json:15]: Don't change this.

File: src/styles/App.scss [hunk 2]
```diff
@@ -9,4 +9,5 @@
   .AppHeading {
     font-size: 1.5em;
   }
+
```
Comment (praveenscience) [inline src/styles/App.scss:12]: Remove this extra line.

File: src/styles/Header.scss
Comment (praveenscience) [inline src/styles/Header.scss:1]: Just have only this file.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Have only one file please.

**Comment by AmanJain18** [comment]
@Shreyans2004 Why u have commited this and raise the pR into main brach.
There is seperate branch for Landing page website. ➡️ `website` branch.
And also why two seperate PR for one issue.
cc @praveenscience  Please Look into this.

**Comment by Shreyans2004** [comment]
Sorry for the inconvenience. I'll take care of it next time.

**Comment by AmanJain18** [comment]
> Referen

No next time, make new pr, as by these no effect would be apply to the website

**Comment by praveenscience** [comment]
@AmanJain18 This can be

**Pull request closed without merging**